etna/metrics/metrics.py

```diff
@@ -1,3 +1,5 @@
+from functools import partial
```

Comment:
How are we going to deal with passing multioutput to kwargs?

Reply:
I don't really think it should be possible to pass `multioutput` to kwargs, it doesn't make sense. Could you elaborate on your comment?